src/Tickets/QR.tsx

```diff
@@ -33,27 +33,8 @@ const QR = ({ activeTab, ticket, redeemKey }: QRProps) => {
     )
   }
 
-  const { ticketId, eventId } = ticket
-  const qrText = prepareQrText(redeemKey, ticketId, eventId)
-  return <QRCode size={200} value={qrText} />
-}
-
-const prepareQrText = (
-  redeemKey: string,
-  ticketId: string,
-  eventId: string
-) => {
-  const scanObject = {
-    type: 0,
-    data: {
-      redeem_key: redeemKey,
-      id: ticketId,
-      event_id: eventId,
-      extra: '',
-    },
-  }
-
-  return JSON.stringify(scanObject)
+  const { ticketId } = ticket
+  return <QRCode size={200} value={JSON.stringify({ ticketId, redeemKey })} />
```

Comment:
We don't need the `ticketId` at all, the redeem endpoint is now
`POST /events/{id}/redeem` `{"redeem_key": "key"}`
```suggestion
  return <QRCode size={200} value={JSON.stringify({ redeemKey })} />
```